src/components/VideoList/VideoList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import VideoCard from '../VideoCard';
+
+const VideoListContainer = styled.div`
+  text-align: center;
+`;
+
+export default function VideoList({ videos, handleVideoSelected }) {
```

Comment:
Instead of using prop drilling with `handleVideoSelected` use the context API from react